docs/spec.md

```diff
@@ -2263,23 +2263,58 @@ Computes dot products between slices of `lhs` and slices of `rhs` and produces a
 More formally, `result[result_index] = dot_product`, where:
 
 <!-- markdownlint-disable line-length -->
-* `lhs_result_dimensions = [d for d in axes(lhs) and d not in lhs_batching_dimensions and d not in lhs_contracting_dimensions]`.
+* `If lhs uses non-quantized tensor and rhs uses quantized tensor:`
+
+  ```python
+   for each batch_size in [0, dim(lhs, lhs_batching_dimensions[0])]:
```

Comment:
Is this `batch_size` or `batch_idx`?

Also, what are you iterating for?  Do you expect `batch_size` to be `0` and then `dim(lhs, lhs_batching_dimensions[0])`?  Or do you expect `batch_size` to be `0`, `1`, `2`, ..., `dim(lhs, lhs_batching_dimensions[0]) - 1`?  If it is later, please just write python code:

```
  for batch_size in range(0, dim(lhs_batching_dimensions[0]):
    ...
```

Reply:
>> Or do you expect batch_size to be 0, 1, 2, ..., dim(lhs, lhs_batching_dimensions[0]) - 1
Yes, that is what I expect. 

I will do the make the suggested in the next commit.